.github/workflows/run_server.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# Please note - this command is meant to be run from two directories above where it currently lives.
+
+if ! command -v tmux &> /dev/null
+then
+    echo "tmux not installed, trying to install for ubuntu test environment"
+    sudo apt install tmux
+fi
+
+if ! command -v tmux &> /dev/null
+then
+    echo "tmux could not be found, aborting"
+    exit 1
+fi
+
+echo "tmux installed, setting eula and starting server"
+
+mkdir -p ./run
+echo "eula=true" > ./run/eula.txt
+
+tmux new-session -d
+echo "Starting Minecraft Server"
+tmux send-keys './gradlew runServer' C-m
+
+# Give the server 90 seconds to start to generate a world file.
+# If we start getting errors where it doesn't start fast enough, might need to raise this value
+# This takes <30 seconds on my box, but GitHub is really slow.
+sleep 90
+
+echo "Killing session"
+tmux kill-session
+
+echo "Printing server latest log:"
+cat ./run/logs/latest.log
+
+# GitHub is really slow so we're just going to check it generated chunks...
+echo "Checking for successful chunkloading (\"Preparing spawn area: \" messages)"
+SPAWN_AREA=$(grep "Preparing spawn area: " < ./run/logs/latest.log | wc -l)
+
+if [ "$SPAWN_AREA" != "0" ]; then
+    echo "Server started successfully! :)"
+    # Good enough.
```

Comment:
I can think of a number of crash-after-loads where this _isn't_ good enough, but looking for errors is fruitless because of how many of them are logged after being caught, so...

You know what, sure, good enough :P

Reply:
So the reason I didn't try doing that is 1) how the heck am I going to run a Minecraft _client_ in a GL-less context where I barely have access to `sudo` and 2) GitHub is so slow that world generation can take probably upwards of 5-10 minutes and I don't want to wait that long when most errors will be found in the first 30 seconds of the classloader.